pkg/porter/testdata/show/expected-output.txt

```diff
@@ -10,13 +10,13 @@ Bundle:
 Labels:
   io.cnab/app: wordpress
   io.cnab/appVersion: v1.2.3
-
```

Comment:
Please undo the changes to the output formatting so that the display of the expected outputs rename unchanged (unless we are showing new/different data, the whitespace and headers should not change).